tests/data/samplesheets/categorize/basic.csv

```diff
@@ -0,0 +1,7 @@
+sample,sample_name,metadata_1,metadata_2,metadata_3,metadata_4,metadata_5
+sample1,"ABC",homo sapiens,human,,,
+sample2,"DEF",,dog,,,
+sample3,"GHI",,,eggs,,
+sample4,"JKL",,,,farm,wastewater
+sample5,"MNO",,,,,
+sample6,"PQR",homo sapiens,dog,,,
```

Comment:
Recommendations for more tests:

Rows in the same sample sheet:
- homo sapiens, but missing human
- homo sapiens (not all lowercase), but missing human
- human, but missing homo sapiens
- human (not all lowercase), but missing homo sapiens
- something in every column
- non-human/homo sapiens host and food product
- food product and environmental site
- food product and environmental material

Different sample sheet:
- columns in a different order